### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Hmamouche** [rbagd/dynfactoR#1]
Hi,

I'm interested in Dynamic Factor Models, but I didn't find any implementation in R except on this package, but I didn't find the manual that contains the description of the functions of this package.

Thank you for your help,

Youssef

**Comment by rbagd** [comment]
I agree. I've been working recently on a complete rewrite of all the package material but I have no ETA on that yet.

Most of the functionality should be more or less familiar to people with knowledge of the academic work on these models. I may suggest however to look into the [original Matlab files](https://dataverse.harvard.edu/dataset.xhtml?persistentId=hdl:1902.1/21827) from Doz, Gianone & Reichlin. They are a little more documented at this time.

**Comment by jetryt** [comment]
Hi,

Thank you for your efforts on this package.

Could please provide any examples that use this package, I'm not sure how to use it.

Thank you very much.

**Comment by saulst** [comment]
Hi, 
I'm working with dynamic factor models and I found your package. I want to know if with your package I can get any proof that the calculated factors are significant
